diff --git a/setup.cfg b/setup.cfg
index 9689996..8ee7280 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -9,30 +9,30 @@ long_description = file: README.md
 long_description_content_type = text/markdown
 url = https://github.com/jo-mueller/napari-stl-exporter.git
 classifiers = 
-	Development Status :: 3 - Alpha
-	Intended Audience :: Developers
-	Framework :: napari
-	Topic :: Software Development :: Testing
-	Programming Language :: Python
-	Programming Language :: Python :: 3
-	Programming Language :: Python :: 3.7
-	Programming Language :: Python :: 3.8
-	Programming Language :: Python :: 3.9
-	Operating System :: OS Independent
-	License :: OSI Approved :: BSD License
+    Development Status :: 3 - Alpha
+    Intended Audience :: Developers
+    Framework :: napari
+    Topic :: Software Development :: Testing
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Operating System :: OS Independent
+    License :: OSI Approved :: BSD License
 
 [options]
 packages = find:
 python_requires = >=3.7
 package_dir = 
-	=src
+    =src
 install_requires = 
-	napari
-	napari-plugin-engine>=0.1.4
-	numpy
-	scikit-image
-	vedo
-	npe2
+    napari
+    napari-plugin-engine>=0.1.4
+    numpy
+    scikit-image
+    vedo
+    npe2
 include_package_data = True
 
 [options.packages.find]
@@ -40,13 +40,12 @@ where = src
 
 [options.package_data]
 napari-stl-exporter = 
-	napari.yaml
+    napari.yaml
 
 [options.entry_points]
 napari.manifest = 
-	napari-stl-exporter = napari_stl_exporter:napari.yaml
+    napari-stl-exporter = napari_stl_exporter:napari.yaml
 
 [egg_info]
 tag_build = 
 tag_date = 0
-
